mantidproject/mantid
# LoadPSIMuonBin: Max array value is chosen when each spectra has individual T0

**Description of work.**
PSI data is provided with a T0 for each spectrum. T0 is now set as the max value in the array instead of the first value.

**To test:**
1. Go to Interfaces -> Muon -> Muon Analysis 2
2. Click browse and load a PSI data file, e.g. ```deltat_tdc_dolly_1529.bin```
3. In the workspace group which loads, view the raw data sample logs.
4. Look for the integerT0 values and make note of the highest value.
5. Double click the raw data workspace and in the table scroll along to the column of the highest value from step 4.
6. Check that the time is the same as the Time Zero in the Muon Analysis window.

Refs #26496 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
